### PR TITLE
feat list picadillo created

### PR DESCRIPTION
**Pull request by abrilmunozc**

### CODE REVIEW

File: 04/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Receta de Picadillo</title>
```
Comment (eamzea) [inline 04/index.html:6]: Where are the others meta tags we have already seen?

File: 04/index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    </ul>
+    <h2>Instrucciones para cocinar</h2>
+    <ol>
+      <li>Calienta el aceite, fríe la cebolla hasta que esté transparente</li>
+      <li>Agrega la carne y cocina por 5 minutos moviendo ocasionalmente</li>
+      <li>Licúa los jitomates con el ajo, el Concentrado de Tomate con Pollo CONSOMATE® y los chiles</li>
+      <li>Vierte sobre la carne, tapa y cocina hasta que comience a hervir.</li>
+      <li>Agrega las papas, las zanahorias, los chícharos y el laurel</li>
+      <li>Cocina por 5 minutos más y ofrece</li>
+    </ol>
```
Comment (eamzea) [inline 04/index.html:37]: Where are the containers in this part of the code?

File: 04/index.html (hunk 1 above)
Comment (eamzea) [inline 04/index.html:6]: Where are the others meta tags we have already seen?

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

File: 04/index.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  <meta name="description" content="Ingredientes secretos y fórmula exacta para preparar Picadillo">
+  <meta name="auhtor" content="Abril Muñoz">
+  <title>Receta de Picadillo</title>
+</head>
+<body>
+  <main>
+    <header>
+      <p>
+        <h1>Picadillo</h1>
+      </p>
```
Comment (eamzea) [inline 04/index.html:16]: A heading tag inside a paragraph tag?

File: 04/index.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+      <li>5 Jitomate cortados en cuartos</li>
+      <li>1 Diente de ajo</li>
+      <li>2 Cubos de Concentrado de Tomate con Pollo CONSOMATE®</li>
+      <li>2 Chiles chipotle adobados</li>
+      <li>2 Papas cortadas en cubos y cocidas</li>
+      <li>2 Zanahorias cortadas en cubos y cocidas</li>
+      <li>1/2 Taza de chícharos cocidos</li>
+      <li>3 Hojas de laurel</li>
+    </ul>
+    </article>
```
Comment (eamzea) [inline 04/index.html:33]: Indentation

File: 04/index.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+      <h2>Instrucciones para cocinar</h2>
+    <ol>
+      <li>Calienta el aceite, fríe la cebolla hasta que esté transparente</li>
+      <li>Agrega la carne y cocina por 5 minutos moviendo ocasionalmente</li>
+      <li>Licúa los jitomates con el ajo, el Concentrado de Tomate con Pollo CONSOMATE® y los chiles</li>
+      <li>Vierte sobre la carne, tapa y cocina hasta que comience a hervir.</li>
+      <li>Agrega las papas, las zanahorias, los chícharos y el laurel</li>
+      <li>Cocina por 5 minutos más y ofrece</li>
+    </ol>
+    </section>
```
Comment (eamzea) [inline 04/index.html:44]: Indentation

File: 04/index.html [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    </article>
+    <section>
+      <h2>Instrucciones para cocinar</h2>
+    <ol>
+      <li>Calienta el aceite, fríe la cebolla hasta que esté transparente</li>
+      <li>Agrega la carne y cocina por 5 minutos moviendo ocasionalmente</li>
+      <li>Licúa los jitomates con el ajo, el Concentrado de Tomate con Pollo CONSOMATE® y los chiles</li>
+      <li>Vierte sobre la carne, tapa y cocina hasta que comience a hervir.</li>
+      <li>Agrega las papas, las zanahorias, los chícharos y el laurel</li>
+      <li>Cocina por 5 minutos más y ofrece</li>
```
Comment (eamzea) [inline 04/index.html:42]: If every step finishes without or with a dot, all steps should finish the same way

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

**Review by soyedgarzea — APPROVED** [review]

**Pull request merged**